### PR TITLE
Create issue template for deployments

### PR DESCRIPTION
**Pull request by jessepinho**
Previously, the deployment docs pointed to a link that had the issue template embedded in the query string, which made it hard to update. This PR extracts that template to an issue template markdown file, and updates the link. Note that this can only really be tested once it's merged, since the link in the deployment docs points to a template that won't exist until this is merged.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (jessepinho) [inline .github/ISSUE_TEMPLATE/deployment.md:16]: @grod220 does something need to be committed after running `pnpm changeset version`? If so, does that require a separate PR that must be reviewed and merged before continuing the deployment?

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (jessepinho) [inline .github/ISSUE_TEMPLATE/deployment.md:19]: @grod220 these require being committed to a PR branch, then reviewed and merged, right?

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (jessepinho) [inline .github/ISSUE_TEMPLATE/deployment.md:21]: After https://github.com/penumbra-zone/web/issues/857, we'll have a script to take care of the zipping

File: docs/deployment.md
Comment (grod220) [inline docs/deployment.md:5]: Nice! Didn't know you could create templates like this.

**Review by grod220 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (hdevalence) [reply .github/ISSUE_TEMPLATE/deployment.md:24]: I feel like this could be out of scope for Github planning and something to revisit post repo-disentangling

Comment (grod220) [reply .github/ISSUE_TEMPLATE/deployment.md:24]: All the stuff related to the extension will be migrated. And in the new repo, these deployment docs are still relevant.

Comment (grod220) [reply .github/ISSUE_TEMPLATE/deployment.md:24]: We'll just need to amend a few lines and file locations. But think it's best not to lose this work waiting on the migration.

Comment (hdevalence) [reply .github/ISSUE_TEMPLATE/deployment.md:24]: Sounds good, to clarify my comment was along the lines of "here's reasons why this can be imperfect now" rather than "we should not do this at all"

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (jessepinho) [reply .github/ISSUE_TEMPLATE/deployment.md:15]: OK, got it. Are we no longer ignoring this, then?

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (jessepinho) [reply .github/ISSUE_TEMPLATE/deployment.md:19]: OK. Outside the scope of this PR, but we should discuss deployment workflows soon. If the person running a deployment has to make two separate PRs (one for this, and one for changesets above), AND wait for approval on them, deployments will be a huge pain.

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (grod220) [reply .github/ISSUE_TEMPLATE/deployment.md:15]: Yes, think we good to add this

File: .github/ISSUE_TEMPLATE/deployment.md
Comment (jessepinho) [reply .github/ISSUE_TEMPLATE/deployment.md:16]: Cool, done

**Comment by grod220** [comment]
oooo PR 888 nice

**Pull request merged**
